--- rogue.asm	1111-11-11 11:11:11.111111111 -1111
+++ rogue_notraps.hak	1111-01-11 11:11:11.111111111 -1111
@@ -338,7 +338,7 @@
         ; Aaaarghhhh!
         ;
 trap_found:
-        call random6            ; Random 1-6
+        ret                     ; Trap does nothing!
 sub_hp: neg ax                  ; Make it negative
 add_hp: add ax,[bp+hp]          ; Add to current HP
     %ifdef com_file
